components/fxa-client/examples/devices_api.rs

```diff
@@ -4,7 +4,7 @@
 
 use cli_support::prompt::prompt_string;
 use dialoguer::Select;
-use fxa_client::{device, Config, FirefoxAccount, IncomingDeviceCommand};
+use fxa_client::{device, ffi::MetricsParams, Config, FirefoxAccount, IncomingDeviceCommand};
```

Comment:
Feel a little awkward access this using the ffi module 🙃 Another option is to make `oauth` public